_posts/2024-12-28-openloco-v24.12.md

```diff
@@ -0,0 +1,104 @@
+---
+title:      "OpenLoco version 24.12"
+tagline:    "Release blog post"
+author:     "Aaron van Geffen"
+date:       2024-11-29 12:00:00 +0100
+layout:     single
+classes:    wide
+categories: news
+
+---
+
+OpenLoco v24.12 is out! This release mainly focuses on bug fixes,
+but there's some exciting changes under the hood as well!
+
+For a complete summary of changes, please find the
+[OpenLoco changelog](https://github.com/OpenLoco/OpenLoco/releases/tag/v24.12) on GitHub.
+
+## Jukebox rework
+
+The Locomotion jukebox features various tracks for the game's different eras. These can be played
+in their original order, but you can also make your own playlist. There was an edge case, though,
+where a playlist with your one favourite track would stop playing after that one track!
+Fed up with this, new contributor @LeeSpork removed the cause in
+[#2750](https://github.com/OpenLoco/OpenLoco/pull/2750). Moreover, he refactored the jukebox code,
+so it now lives in its own compilation unit in [#2771](https://github.com/OpenLoco/OpenLoco/pull/2771).
+Nice work!
+
+## Paint interop removal
+
+The game's various drawing routines for rendering the world viewport, called paint routines,
+have all been implemented as of last most. This marked a huge milestone for the project, and while it
```

Comment:
"last most"? did you mean something like "last month"

Reply:
Nice catch, that's a solid typo alright
```suggestion
have all been implemented as of last month. This marked a huge milestone for the project, and while it
```